XMonad/Prompt/XMonad.hs

```diff
@@ -38,10 +39,10 @@ import XMonad.Prelude (fromMaybe)
 -- For detailed instruction on editing the key binding see
 -- "XMonad.Doc.Extending#Editing_key_bindings".
 
-data XMonad = XMonad
+newtype XMonad = XMonad String
 
 instance XPrompt XMonad where
-    showXPrompt XMonad = "XMonad: "
+    showXPrompt (XMonad str) = str <> ":"
```

Comment:
There should probably be a space at the end of the string, just like before, right?

Reply:
Oops, I'll fix that right away